### PR TITLE
finishes pQueue and fixes formatting and tests

### PR DESCRIPTION
**Pull request by lizzkats**

### CODE REVIEW

File: spec/priorityQueue.js [hunk 1]
```diff
@@ -3,29 +3,57 @@ import chaiChange from 'chai-change'
...
-      const pQueue = new priorityQueue
-    expect(() => pQueue.enqueue('pizza', 90))
-    .to.alter(() => pQueue.length(), { from: 0, to: 1 })
+      const newQueue = new priorityQueue
+    expect(() => newQueue.enqueue('pizza', 90))
+    .to.alter(() => newQueue.length(), { from: 0, to: 1 })
     })
       const newQueue = new priorityQueue
-      console.log(newQueue)
       newQueue.enqueue('pizza', 90)
```
Comment (deonna) [inline spec/priorityQueue.js]: A better test would be to add the lowest priority item first and a higher priority second, and then an item that is in between third. 

What to ask yourself when you're writing tests: could your tests pass even if you implemented your functionality incorrectly? In this case, if I just pushed the items onto an array without considering `priority`, I'd get the same result of `pizza` coming first and `potato` coming second.

File: spec/priorityQueue.js [hunk 2]
```diff
@@ -3,29 +3,57 @@ import chaiChange from 'chai-change'
...
+  context('back()', () => {
+    it('returns null if the queue is empty', () => {
+      expect(emptyQueue.back()).to.equal(null)
+    })
+    it('returns the last item of the queue', () => {
+      expect(pQueue.back()).to.equal('sock')
+    })
+  })
+  context('isEmpty()', () => {
+    it('returns true if the array is empty', () => {
```
Comment (deonna) [inline spec/priorityQueue.js]: Add test for if the array is not empty. Make sure that both cases are unit tested.

File: spec/priorityQueue.js [hunk 3]
```diff
@@ -3,29 +3,57 @@ import chaiChange from 'chai-change'
...
+    pQueue.enqueue('sock', 100)
   })
 
+
   context('enqueue()', () => {
     it('adds an element to the queue' , () => {
-      const pQueue = new priorityQueue
-    expect(() => pQueue.enqueue('pizza', 90))
-    .to.alter(() => pQueue.length(), { from: 0, to: 1 })
+      const newQueue = new priorityQueue
```
Comment (deonna) [inline spec/priorityQueue.js]: You may want to reconsider using the simplified grammar to instantiate a new instance of priorityQueue. It's better signaling to include `()`, even if the constructor takes no arguments.

Additionally, the convention for classes is to capitalize the first letter of the name of the class.

File: spec/stack.js [hunk 4]
```diff
@@ -5,55 +5,48 @@ import Stack from '../src/stack'
...
-      it('returns the number of elements in the stack.', () => {
-        const myStack = new Stack()
-        myStack.push('dog')
-        myStack.push('chicken')
+  context('peek()', () => {
+      it('shows top element in the stack', () => {
+      const myStack = new Stack()
+      myStack.push('potato')
+      myStack.push('sock')
+      expect(myStack.peek()).to.eql('potato')
```
Comment (deonna) [inline spec/stack.js]: Stack is LIFO/last in first out. If you push on `potato` and then push on `sock`, peek should return the element at the top, which is `sock`.

File: spec/stack.js [hunk 5]
```diff
@@ -5,55 +5,48 @@ import Stack from '../src/stack'
...
 
-    context('pop()', () => {
-      it('returns and removes the top element in the stack', () => {
-        const myStack = new Stack()
-        myStack.push('cookie')
-        myStack.push('ice cream')
-        expect(() => myStack.pop())
-          .to.alter(() => myStack.length(), { from: 2, to: 1 })
+  context('pop()', () => {
+    it('returns and removes the top element in the stack', () => {
```
Comment (deonna) [inline spec/stack.js]: Also check the value of the element that is popped.
